docs/getting-started/inferno-cli.md

```diff
@@ -64,4 +65,64 @@ using the `-a` option, and set the name, summary and description values to your
 You should manually inspect this file if you want to [publish or distribute the Test Kit](/docs/distributing-tests.html).
 
 
-The `--implementation-guide` or `-i` option will look at an absolute file path if its prefixed by `file:///` or lookup the name in the [official FHIR package registry](https://packages.fhir.org/).
\ No newline at end of file
+The `--implementation-guide` or `-i` option will look at an absolute file path if its prefixed by `file:///` or lookup the name in the [official FHIR package registry](https://packages.fhir.org/).
+
+## Running a Test Kit in Command Line
```

Comment:
I think we need to rethink where some of this content lives now that the 'inferno cli' does more than just serve 'writing test kits' but also is important in running them.  But let's leave it here for now!